### PR TITLE
Milestone 5: Database performance audit

### PR DESCRIPTION
**Pull request by Xghayor**
In this milestone we completed:
1.Analyzing database performance
2.Improved query execution time

BEFORE:

Screenshot 2023-10-03 105042

AFTER:

Screenshot 2023-10-03 105925

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -283,3 +283,16 @@ GROUP BY species.name, vets.name
...
+-- Try them on your machine to confirm it:
+
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits
+WHERE animals_id = 4;
+
+EXPLAIN ANALYZE SELECT * FROM visits 
+WHERE vet_id = 2;
+
+EXPLAIN ANALYZE SELECT * FROM owners 
+WHERE email = 'owner_18327@mail.com';
```
Comment (thecodechaser) [inline queries.sql:298]: - [ ] Looking at the screenshot for the first and third query before adding an index, the Planning and Execution time is below 1000ms. According to the requirement, you need to keep populating the database with the data provided until the execution time is above 1000ms, and then you work on it to reduce it. Please make sure to fix this issue.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Xghayor, and @vikas924👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Good job with the performance audit. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Xghayor) [reply queries.sql:298]: @thecodechaser 
Thanks for the feedback, we will fix it

**Comment by Xghayor** [comment]
Updated Screen shots:

BEFORE: 



AFTER:

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @Xghayor and @vikas924 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- Good job with the performance audit :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**